test/acceptance/admin_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Community.Acceptance.AdminTest do
+  use Community.AcceptanceCase
+
+  test "it allows admins to see the jobs" do
+    job = create(:job)
+
+    navigate_to "/admin/jobs"
+
+    assert flash_text =~ "You must log in to see this page"
+
+    navigate_to "/session/new"
```

Comment:
Should this acceptance test just assume that the user is already redirected to the login page. Or, is that not something that should happen?